notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_optimized.ipynb

```diff
@@ -648,17 +648,22 @@
         "    optimized=feature_online_store_pb2.FeatureOnlineStore.Optimized(),\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    FEATURE_GROUP_ID = "product_features"  # @param {type: "string"}
</pre><p>Use "product_features_unique" for the EATURE_GROUP_ID&nbsp;to avoid naming collision. The integration test framework will automatically replace _unique with an unique id when executing the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>